clouddriver/src/test/java/io/armory/plugin/eap/loaders/URLCredentialsLoaderTest.java

```diff
@@ -44,138 +46,182 @@ public void setUp() {
     }
 
     @Test
     public void testReplaceNotDefinedEnvVars() {
-        URLCredentialsLoader<KubernetesConfigurationProperties.ManagedAccount> loader = new URLCredentialsLoader<>(
+        URLCredentialsLoader<KubernetesAccountProperties.ManagedAccount> loader = new URLCredentialsLoader<>(
                 null,
                 EAPConfigurationProperties.FileFormat.YAML,
-                KubernetesConfigurationProperties.ManagedAccount.class,
+                KubernetesAccountProperties.ManagedAccount.class,
                 secretManager) {
             @Override
             protected InputStream getInputStream() {
                 return URLCredentialsLoaderTest.class.getResourceAsStream("/single-not-env.yml");
             }
         };
 
-        List<KubernetesConfigurationProperties.ManagedAccount> actual = loader.getCredentialsDefinitions();
+        List<KubernetesAccountProperties.ManagedAccount> actual = loader.getCredentialsDefinitions();
         assertEquals("${UNKNOWN}", actual.get(0).getName());
     }
 
+    @Test
+    public void testCloudDriverAccounts() {
+        URLCredentialsLoader<KubernetesAccountProperties.ManagedAccount> cdl = new URLCredentialsLoader<>(
+                null,
+                EAPConfigurationProperties.FileFormat.YAML,
+                KubernetesAccountProperties.ManagedAccount.class,
+                secretManager) {
+            @Override
+            protected InputStream getInputStream() {
+                return URLCredentialsLoaderTest.class.getResourceAsStream("/clouddriver-mixed.yml");
+            }
+        };
+        List<KubernetesAccountProperties.ManagedAccount> cda = cdl.getCredentialsDefinitions();
+        assertTrue(cda.size() == 1 && "kube".equals(cda.get(0).getName()));
+    }
+
+    @Test
+    public void testCloudFoundryAccounts() {
+        URLCredentialsLoader<CloudFoundryConfigurationProperties.ManagedAccount> cfl = new URLCredentialsLoader<>(
+                null,
+                EAPConfigurationProperties.FileFormat.YAML,
+                CloudFoundryConfigurationProperties.ManagedAccount.class,
+                secretManager) {
+            @Override
+            protected InputStream getInputStream() {
+                return URLCredentialsLoaderTest.class.getResourceAsStream("/cf-multiple.yml");
+            }
+        };
+        List<CloudFoundryConfigurationProperties.ManagedAccount> cfa = cfl.getCredentialsDefinitions();
+        assertTrue(cfa.size() == 2 && "cf".equals(cfa.get(0).getName()) && "cf".equals(cfa.get(1).getName()));
+        cfl = new URLCredentialsLoader<>(
+                null,
+                EAPConfigurationProperties.FileFormat.YAML,
+                CloudFoundryConfigurationProperties.ManagedAccount.class,
+                secretManager) {
+            @Override
+            protected InputStream getInputStream() {
+                return URLCredentialsLoaderTest.class.getResourceAsStream("/clouddriver-mixed.yml");
```

Comment:
On this test case use only cloudfoundry accounts, not the mixed kube/cf file, so that the test only loads `cf-multiple.yml` and asserts that there are two accounts. We need to change the name of one of those two accounts (in `cf-multiple.yml`) so that they have different names.